app/src/main/java/com/codrut/newsreader/view/bindings/VisibilityBinding.java

```diff
@@ -0,0 +1,16 @@
+package com.codrut.newsreader.view.bindings;
+
+import android.view.View;
+import android.widget.ProgressBar;
+
+import androidx.databinding.BindingAdapter;
+import androidx.databinding.ObservableBoolean;
+
+public class VisibilityBinding {
```

Comment:
We can delete this class and keep the one from previous comment, ViewBindings. It's better suited to group binding adapter to the view types it applies as it tends to group the extension of functionality in one place, such as TextViewBindings will depict what new properties we could be using when working with TextViews